CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.12.0
+
+### Changed
+
+- Fixed a bug that allowed `ComponentDef` to be captured in closures for a long
+  time, which could lead to using stale values in complex scenarios where
+  `ComponentDef` is not constant, but depends on arguments. See [#83](https://github.com/collegevine/purescript-elmish/pull/83).
```

Comment:
Nit: Within the same repo, this syntax will autolink AFAIK 👍 

```suggestion
  `ComponentDef` is not constant, but depends on arguments. See #83.
```

Reply:
No, it doesn't. I tried this first.